tests/FSharp.Compiler.ComponentTests/FSharpChecker/TransparentCompiler.fs

```diff
@@ -800,4 +800,116 @@ module Stuff =
 
         //Assert.Equal<string>(hash, hash2)
 
-        ()
\ No newline at end of file
+        ()
+
+let mkWorkflowBuilderForResponseFile
```

Comment:
This is pretty universal, I think we should move it to `ProjectGeneration.fs`. That way it's also available for benchmarks. 

I would probably also separate creating the project from creating the builder.

Reply:
Moved it.